web/themes/custom/asulib_barrio/scss/components/views.scss

```diff
@@ -194,18 +197,35 @@
             justify-content: center;
         }
     }
+    span.views-summary {
```

Comment:
do we need all this views-summary styling or does the pagination stuff cover it?

Reply:
Probably. I don't know. I'll just remove.